### DISCUSSION TITLE
【slug优化】按规则自动生成文章短路径

### DISCUSSION THREAD

**Discussion by tangly1024** [tangly1024/NotionNext#2012] [Ideas]
**为什么提出这个新的特性改动**
来自群友的反馈

**描述一下你推荐的解决方案**
按照标题自动生成：
拼音路径；类似
> Windows系统更改Mac地址
Windows-xi-tong-geng-gai-Mac-di-zhi

> Windows10 2022 镜像包V1.3.1
Windows10-2022-jing-xiang-bao-V1.3.1

**Comment by tangly1024** [comment]
新版本中，支持按照年月日的格式生成链接：
https://github.com/tangly1024/NotionNext/blob/c4f0bbddb100d79c47e8330d904b72dd2bd6e863/blog.config.js#L70-L74

**Comment by aistart** [comment]
这个文档序号的需求挺实用的。但是这种“年月日”的方案，导致文档编号并不一定是唯一的。是否能修改为直接用唯一的系统时间戳？

**Comment by tangly1024** [comment]
> 这个文档序号的需求挺实用的。但是这种“年月日”的方案，导致文档编号并不一定是唯一的。是否能修改为直接用唯一的系统时间戳？

用系统时间戳，页面路径可读性就很差，和用notion自带的id似乎没有差别。
